axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/ProjectTaskReportingValuesComputingServiceImpl.java

```diff
@@ -322,33 +322,36 @@ protected void computeLandingValues(
     }
   }
 
-  protected BigDecimal computeRealCosts(ProjectTask projectTask, Project project)
-      throws AxelorException {
+  protected BigDecimal computeRealCosts(
+      ProjectTask projectTask, Project project, boolean unitIsTimeUnit) throws AxelorException {
     BigDecimal unitCost = computeUnitCost(projectTask, project);
     projectTask.setUnitCost(unitCost);
 
     BigDecimal timeSpent = getTaskSpentTime(projectTask);
 
-    BigDecimal realCost =
-        timeSpent
-            .multiply(unitCost)
-            .setScale(RESULT_SCALE, RoundingMode.HALF_UP)
-            .add(
-                projectTask.getPurchaseOrderLineList().stream()
-                    .filter(
-                        purchaseOrderLine -> {
-                          Integer purchaseOrderStatus =
-                              purchaseOrderLine.getPurchaseOrder().getStatusSelect();
-                          return purchaseOrderStatus == PurchaseOrderRepository.STATUS_VALIDATED
-                              || purchaseOrderStatus == PurchaseOrderRepository.STATUS_FINISHED;
-                        })
-                    .map(PurchaseOrderLine::getExTaxTotal)
-                    .reduce(BigDecimal::add)
-                    .orElse(BigDecimal.ZERO));
+    BigDecimal realCost = BigDecimal.ZERO;
+
+    if (unitIsTimeUnit) {
```

Comment:
I think it's better to call again "isTimeUnitValid" than to add a boolean as argument here